packages/grid/x-data-grid/src/hooks/features/rows/gridRowsSelector.ts

```diff
@@ -8,6 +8,8 @@ export const gridRowCountSelector = createSelector(
   (rows) => rows.totalRowCount,
 );
 
+export const gridLoadingSelector = createSelector(gridRowsStateSelector, (rows) => rows.loading);
```

Comment:
If it is in the row state, should it be `gridRowsLoadingSelector` ?

Reply:
Renamed.